man/systemd.mount.xml

```diff
@@ -76,13 +76,13 @@
     which define the way the processes are terminated, and in
     <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
     which configure resource control settings for the processes of the
-    service. Note that the User= and Group= options are not
-    particularly useful for mount units specifying a
-    <literal>Type=</literal> option or using configuration not
-    specified in <filename>/etc/fstab</filename>;
+    service. Note that the User= and Group= options are not useful for
+    mount units. Systemd always passes two values to
```

Comment:
we try not to say "systemd" as a generic name, as that's the whole suite. Please say "The service manager" or "The system and service manager" or so...

Reply:
s/two values/two parameters/

Reply:
Changing to "parameters" sounds fine to me.  (I have read that functions are declared with parameters, and called with arguments; I was also fudging "values" v.s. "options", i.e. we pass two values plus the mount options).

Neither of the suggested phrasings for PID 1 read very well, and "systemd" is more consistent with mentions through the rest of the man page.  I am not rephrasing the manpage as a whole.  Nor do I think it would work very well to skate around by using the passive voice, e.g. saying "Two parameters are passed to mount:".  I could drop the "always" from "Systemd always", to help avoid making the statement too broad.

Reply:
Well, most importantly: thou shalt not write systemd as Systemd... not even at the beginning of sentences. ;-)

Reply:
:D.  Right, _that_ wasn't consistent with the vast majority of other uses.  Amended.